### PR TITLE
editorial: s/fields/members + xref fixes (closes #424)

### LINKED ISSUE #424: Spec talks about dictionary fields instead of dictionary members
Should be changed and linked to dictionary members.

### PR DESCRIPTION
**Pull request by marcoscaceres**
* includes a few markup fixes also.

***
[Preview](https://s3.amazonaws.com/pr-preview/w3c/browser-payment-api/members.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/browser-payment-api/a6ecd22..members:5dce5df.html)

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Could someone please give this a quick review. It's small fixes.

File: index.html [hunk 1]
```diff
@@ -1224,7 +1225,7 @@
         enum PaymentShippingType {
           "shipping",
           "delivery",
-          "pickup"
+          "pickup",
```
Comment (zkoch) [inline index.html:1228]: Really? Weird... :)

**Review by zkoch — APPROVED** [review]
LGTM

File: index.html (hunk 1 above)
Comment (marcoscaceres) [reply index.html:1228]: Trailing comma is good :)

**Comment by marcoscaceres** [comment]
Can I just say, @tobie, the preview and diff integration is freaken amazing! Thank you!!!!1!!one! ❤️

**Comment by tobie** [comment]
@marcoscaceres have you seen the HTML diff has keyboard navigation? :)

**Comment by marcoscaceres** [comment]
NO! STOP!

**Pull request merged**